BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -121,7 +121,7 @@ def initialize(attributes)
     # encapsulated CbrainTask in a version switcher class to allow different
     # CbrainTask classes for each tool version.
     # Returns the newly generated CbrainTask subclass.
-    def integrate(register: true, create_tool_config: false, multi_version: false)
+    def integrate(register: true, create_tool_config: true, multi_version: false)
```

Comment:
I am not against that, since it is indeed a bit strange. Although, we could also call it something like "attempt_tool_config_creation", so that someone could still specify not to make one.